jetcd-core/src/test/java/io/etcd/jetcd/impl/ClusterClientTest.java

```diff
@@ -73,7 +73,7 @@ public void testMemberManagement() throws ExecutionException, InterruptedExcepti
         final Client client = Client.builder().endpoints(n1.clientEndpoints()).build();
         final Cluster clusterClient = client.getClusterClient();
 
-        Member m2 = clusterClient.addMember(n2.peerEndpoints())
+        Member m2 = clusterClient.addMember(n2.peerEndpoints(), false)
```

Comment:
we need a test for isLearner = true

Reply:
Tests of `addMember(peerAddrs)`, `addMember(peerAddrs, true)`, `addMember(peerAddrs, false)` has been updated in the latest commit